.claude/commands/pr-review/review-task-workflow.md

```
@@ -0,0 +1,88 @@
+---
+name: review-task-workflow
+description: Execute PR review tasks systematically using reviewtask
+---
+
+You are tasked with executing PR review tasks systematically using the reviewtask tool. 
```

markdownlint-cli2 (0.17.2)

Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)